v4l2py/device.py

```diff
@@ -583,7 +583,7 @@ def set_priority(fd, priority: Priority):
 
 def create_buffer(fd, buffer_type: BufferType, memory: Memory) -> raw.v4l2_buffer:
     """request + query buffers"""
-    return create_buffer(fd, buffer_type, memory, 1)
+    return create_buffers(fd, buffer_type, memory, 1)
```

Comment:
Correct. Plus another error is that it should return the first element of the list instead of a list with one element.
Would you mind fixing that too? 🙏🏽

Reply:
Sure. :-)